packages/web/src/common/store/social/tracks/sagas.ts

```diff
@@ -622,43 +628,60 @@ function* downloadTracks({
 }) {
   const { trackId: parentTrackId } = tracks[0]
   try {
-    const audiusBackendInstance = yield* getContext('audiusBackendInstance')
-    const apiClient = yield* getContext('apiClient')
     const audiusSdk = yield* getContext('audiusSdk')
     const sdk = yield* call(audiusSdk)
     const trackDownload = yield* getContext('trackDownload')
-    let queryParams: QueryParams = {}
+    // let queryParams: QueryParams = {}
 
     const nftAccessSignatureMap = yield* select(getNftAccessSignatureMap)
     const userId = yield* select(getUserId)
     const nftAccessSignature = original
       ? nftAccessSignatureMap[parentTrackId]?.original ?? null
       : nftAccessSignatureMap[parentTrackId]?.mp3 ?? null
 
-    queryParams = (yield* call(getQueryParams, {
-      audiusBackendInstance,
-      sdk,
-      nftAccessSignature,
-      userId
-    })) as unknown as QueryParams
+    // queryParams = (yield* call(getQueryParams, {
+    //   audiusBackendInstance,
+    //   sdk,
+    //   nftAccessSignature,
+    //   userId
+    // })) as unknown as QueryParams
 
-    queryParams.original = original
+    // queryParams.original = original
 
     const files = tracks.map(({ trackId, filename }) => {
-      queryParams.filename = filename
-      return {
-        url: apiClient.makeUrl(
-          `/tracks/${encodeHashId(trackId)}/download`,
-          queryParams
-        ),
-        filename
-      }
+      // queryParams.filename = filename
```

Comment:
this is currently broken and needs to be fixed with a minor discovery node query param change